### ISSUE TITLE
:not selector work wrong

### ISSUE THREAD

**Issue by voskresla** [Connormiha/jest-css-modules-transform#22]
I made example, just run `npm i && npx jest` https://github.com/Connormiha/jest-css-modules-transform/compare/master...voskresla:bug-with-not-selector

With css:
```
.class2, .class3 {
    display: block;
    color: #fff;
}
.class3:not(.class1) {
    color: black
}
```
we expected Object like:
```
{
  "class1": "class1",
  "class2": "class2",
  "class3": "class3",
}
```
but received object is:
```
Snapshot name: `Style with :not selector should import default 1`

    - Snapshot  - 1
    + Received  + 1

      Object {
    -   "class1": "class1",
    +   "class1)": "class1)",
        "class2": "class2",
        "class3": "class3",
      }

      7 | });
```

**Comment by yepninja** [comment]
@Connormiha 
Maybe it is better to use https://www.npmjs.com/package/postcss-selector-parser for parsing rules and collecting classes?
Or is there any reasons why you don't want to use it?

**Comment by Connormiha** [comment]
@yepninja 
Main reason is performance. I prefer reduce count of dependencies and add extended library only when there is a very good reason.

**Comment by yepninja** [comment]
[postcss-selector-parser](https://github.com/postcss/postcss-selector-parser) is the standard library for postcss. There is even [an issue](https://github.com/postcss/postcss/issues/1145) to integrate it natively to postcss.

**Comment by Connormiha** [comment]
Fix in 4.0.1
